src/cython/vapoursynth.pyx

```diff
@@ -1321,6 +1326,12 @@ cdef class VideoFrame(RawFrame):
         s += '\tHeight: ' + str(self.height) + '\n'
         return s
 
+    def get_read_array(self, int index):
+        return self._getitem_internal(index, True)
+    def get_write_array(self, int index):
+        if self.readonly:
+            raise Error('Cannot obtain write array to read only frame')
+        return self._getitem_internal(index)
```

Comment:
Here, if you must insist:
```suggestion
    def get_read_array(self, int plane):
        data = VideoFrame.__getitem__(self, plane)
        return view.memoryview(data, PyBUF_RECORDS_RO)

    def get_write_array(self, int plane):
        data = VideoFrame.__getitem__(self, plane)
        return view.memoryview(data, PyBUF_RECORDS)
```

`PyBUF_RECORDS` and `PyBUF_RECORDS_RO` cimported from `cpython.buffer`.

Reply:
Done.